superbench/benchmarks/micro_benchmarks/dist_inference_cpp/CMakeLists.txt

```diff
@@ -39,6 +39,9 @@ else()
         if(DEFINED ENV{USE_HIPBLAS_COMPUTETYPE})
             set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_HIPBLAS_COMPUTETYPE=1")
         endif()
+        if(DEFINED ENV{USE_HIPBLASLT_TUNING})
+            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DUSE_HIPBLASLT_TUNING=1")
```

Comment:
why we need to define this during build rather than by default?

Reply:
I make it an option because this feature requires ROCm version >= 5.7, see https://github.com/pytorch/pytorch/commit/0e6eee3c898c293efc3f172180e9f4d79cc0b13f#diff-0734027152d1361573f308813199b762c9aecb10d03d46d182347ed285cf9afcR14.

Reply:
But it's not USE_HIPBLASLT_TUNING, I think we should check ROCm version and set HIPBLASLT_TUNING_SUPPORT

Reply:
Revised to checking ROCm version.